### ISSUE TITLE
Use Arrow package for date/time handling instead of standard datetime

### ISSUE THREAD

**Issue by sfermigier** [abilian/abilian-core#11]
More info: http://arrow.readthedocs.org/en/latest/

**Comment by sfermigier** [comment]
Cf. http://arrow.readthedocs.org/en/latest/#humanize

**Comment by sfermigier** [comment]
Or check https://regebro.wordpress.com/2015/07/27/python-datetime-libraries/

**Comment by vindarel** [comment]
Pendulum claims to fix many Arrow quirks and bugs: https://github.com/sdispater/pendulum

**Comment by sfermigier** [comment]
Yeah, this is an old issue, we need to look into the various alternatives.
